doc/release-notes/tapyrus/release-notes-0.3.0.md

```diff
@@ -0,0 +1,64 @@
+Tapyrus version 0.3.0 is now available for download at:
+  https://github.com/chaintope/tapyrus-core/tapyrus-0.3.0/
```

Comment:
The intention of this url is provide url for v0.3.0 tag, right?

If it is, i think this is correct https://github.com/chaintope/tapyrus-core/tree/v0.3.0

Reply:
If we use github release feature, we can get a url for specific release like this.
https://github.com/chaintope/tapyrus-core/releases/tag/v0.3.0

and it is for v0.3.0 release. so that we can use above url.

can you change the url to this?

Reply:
done!